### PR TITLE
Minor correction common_replacement

### PR DESCRIPTION
**Pull request by VanSlanzar**

### CODE REVIEW

File: ui/raidboss/common_replacement.js [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
     de: 'Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?$1 schließt',
-    cn: '距$1被封锁还有',
     fr: 'Fermeture d(?:e|u|es) $1 dans',
     ja: '$1の封鎖まであと',
+    cn: '距$1被封锁还有',
     ko: '15초 후에 $1(이|가) 봉쇄됩니다',
   },
   'is no longer sealed': {
     de: 'öffnet sich erneut',
-    fr: 'Ouverture ',
```
Comment (VanSlanzar) [inline ui/raidboss/common_replacement.js:14]: -  the space was he voluntary

File: ui/raidboss/common_replacement.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ let commonReplacement = {
   },
   'Engage!': {
     de: 'Start!',
-    fr: 'À l\'attaque',
+    fr: 'À l\'attaque !',
```
Comment (VanSlanzar) [inline ui/raidboss/common_replacement.js:48]: Missing exclamation mark, with a space in French

File: ui/raidboss/common_replacement.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/common_replacement.js:14]: I'm not sure that this matters either way.  This is trying to match part of the text when a zone is no longer sealed.

File: ui/raidboss/common_replacement.js (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/common_replacement.js:48]: Sorry, but please don't include the exclamation point here.  See #57 for more details.

File: ui/raidboss/common_replacement.js (hunk 1 above)
Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js:14]: I remove it as a precaution, if that does not create a problem.

File: ui/raidboss/common_replacement.js (hunk 2 above)
Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js]: Wow, this was not easy. 
- Was this option not retained?
engage

Comment (mooondark) [reply ui/raidboss/common_replacement.js]: Y'en a pas besoin car ça parse "À l'attaque" peu importe ce qu'il y a après. On a fait pas mal de test il y a un moment et c'est comme ça que ça fonctionne le mieux :)

Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js]: the correction was made as requested. Why don't you answer me on the discord? 
Thank you for your answers

File: ui/raidboss/common_replacement.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/common_replacement.js:14]: A precaution against what? Sorry, I just don't understand your intention.

Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js:14]: If it is not useful to delete this space, I can cancel this request. Sorry for the difficulty of comprehension.

Comment (quisquous) [reply ui/raidboss/common_replacement.js:14]: It was more just me trying to understand what the goal was or if you had noticed any problems.  It sounds like this is working, and I think it looks fine to me.  I will just close this pull request then, if you don't mind.

**Comment by VanSlanzar** [comment]
Thank you.

**Pull request closed without merging**